pkg/virt-handler/vm_test.go

```diff
@@ -2744,33 +2744,39 @@ var _ = Describe("VirtualMachineInstance", func() {
 		})
 
 		It("should succeed with empty VMI and basic commands", func() {
-			result := isGuestAgentSupported(vmi, basicCommands)
+			result, reason := isGuestAgentSupported(vmi, basicCommands)
 			Expect(result).To(BeTrue())
+			Expect(reason).To(Equal(""))
```

Comment:
IIUC, currently,  `isGuestAgentSupported` should return "This guest agent is supported" when the `result` is `true`, so I think that this should not work. Or maybe am I missing something?
This is valid for all the occurrences below. Thanks

Reply:
You are right, fixed. Thanks.